mne/io/ctf/tests/test_ctf.py

```diff
@@ -197,4 +197,15 @@ def test_read_ctf():
     assert_raises(ValueError, read_raw_ctf,
                   op.join(ctf_dir, ctf_fname_continuous), 'foo')
 
+
+@testing.requires_spm_data
+def test_read_spm_ctf():
+    """Test CTF reader with omitted samples."""
+    data_path = spm_face.data_path()
+    raw_fname = data_path + '/MEG/spm/SPM_CTF_MEG_example_faces1_3D.ds'
+    raw = read_raw_ctf(raw_fname)
+    extras = raw._raw_extras[0]
+    assert_equal(extras['n_samp'], raw.n_times)
+    assert_false(extras['n_samp'] == extras['n_samp_tot'])
```

Comment:
Actually that is tested on the line above ``assert_equal(extras['n_samp'], raw.n_times)``.